app/src/main/java/org/systers/mentorship/remote/datamanager/UserDataManager.kt

```diff
@@ -16,49 +15,32 @@ class UserDataManager {
 
     /**
      * This will call the getVerifiedUsers method of UserService interface
-     * @return an Observable of a list of [User]
      */
-    fun getUsers(): Observable<List<User>> {
-        return apiManager.userService.getVerifiedUsers()
-    }
+    suspend fun getUsers(): List<User> = apiManager.userService.getVerifiedUsers()
```

Comment:
This is nice! Did you know you don't have to declare return type when you use = in Kotlin function?

Reply:
Yeah I know, I just got used to writing like this beacuse it's more explicit.

Thank you, I'll make it even shorter :)